### PR TITLE
Added MUI icons and changed footer to sidenav

### PR DESCRIPTION
**Pull request by noliver-ykc**
Added Mui icons pack 
Changed footer to side navigation 

I tried to rename the files from footer from sidenav but it kept bugging out for some reason so i kept it for now and ill rename it if we have more time later on.

https://github.com/xu-jiach/mercari-build-hackathon-2023/assets/74501096/0333aeb7-a5ed-4665-9953-49743e9f64e1

ignore the lack of error message when i type the password wrong on log in-- i havent pulled that specific pr yet.

### CODE REVIEW

File: frontend/simple-mercari-web/src/components/Header/Header.css [hunk 1]
```diff
@@ -11,7 +11,8 @@ header {
   width: 100px;
 }
 
-.logo{
   width: 7em;
   aspect-ratio: 7/3.5;
-}
\ No newline at end of file
+  padding-left: 5em;
+}
```
Comment (mutokrm) [inline frontend/simple-mercari-web/src/components/Header/Header.css:17]: Maybe you forgot to fix this syntax error here!

**Review by mutokrm — CHANGES_REQUESTED** [review]
Just a minor syntax error!

File: frontend/simple-mercari-web/src/components/Header/Header.css (hunk 1 above)
Comment (noliver-ykc) [reply frontend/simple-mercari-web/src/components/Header/Header.css:17]: hi! Im not sure what the syntax error is? Please let me know

Comment (mutokrm) [reply frontend/simple-mercari-web/src/components/Header/Header.css:17]: I think you're missing an opening bracket, and it didn't compile when I checkout this branch!

Comment (noliver-ykc) [reply frontend/simple-mercari-web/src/components/Header/Header.css:17]: i think it should be fixed now, sorry about that!

**Review by mutokrm — APPROVED** [review]
LGTM!

**Pull request merged**